### PR TITLE
Language headers

### PR DESCRIPTION
**Pull request by vxsx**
This PR makes languages in pagetree clickable to switch view so it shows the page names (and page actions) in correct language.

- [x] changelog

screen shot 2016-05-02 at 18 07 08

"en" is default and picked, view page and edit will go to english version

screen shot 2016-05-02 at 18 08 11

"de" is picked, view page and edit will go to german version (page names still will be in english)

### CODE REVIEW

**Comment by FinalAngel** [comment]
👍

**Comment by mkoistinen** [comment]
LGTM

File: CHANGELOG.txt [hunk 1]
```diff
@@ -21,11 +21,7 @@
   small screens.
 * Removed some unnecessary reloads happening after closing sideframe.
 * CMS handles uninstalled apphooks better
-
-=== 3.2.4 (unreleased) ==
-
-- Fixed regression when page couldn't be copied if CMS_PERMISSION was False
-- CMS handles uninstalled apphooks better
+* Added the UI for previewing a pagetree in specific language
```
Comment (czpython) [inline CHANGELOG.txt]: here need to fix this in next pr

File: cms/templates/admin/cms/page/tree/menu.html [hunk 2]
```diff
@@ -260,7 +260,7 @@
     '
     >
 
-    {% language preview_language %}
+    {% language LANGUAGE_CODE %}
```
Comment (czpython) [inline cms/templates/admin/cms/page/tree/menu.html]: So far the CMS has supported changing the display title of the page via this ``?language`` parameter, this change effectively reverts that. I don't agree on breaking this backwards compatibility.

**Comment by FinalAngel** [comment]
LGTM

**Comment by czpython** [comment]
Ok to merge

**Pull request merged**